src/arkode/arkode_adapt.c

```diff
@@ -136,10 +136,18 @@ int arkAdapt(void* arkode_mem, ARKodeHAdaptMem hadapt_mem,
   }
   if (h_cfl <= ZERO)  h_cfl = RCONST(1.0e30) * SUNRabs(hcur);
 
-#if SUNDIALS_LOGGING_LEVEL >= SUNDIALS_LOGGING_INFO
-  SUNLogger_QueueMsg(ARK_LOGGER, SUN_LOGLEVEL_INFO,
-                     "ARKODE::arkAdapt", "new-step-before-bounds",
-                     "h_acc = %"RSYM", h_cfl = %"RSYM, h_acc, h_cfl);
+#if SUNDIALS_LOGGING_LEVEL >= SUNDIALS_LOGGING_DEBUG
+  SUNLogger_QueueMsg(ARK_LOGGER, SUN_LOGLEVEL_DEBUG,
+                     "ARKODE::arkAdapt", "error-history",
+                     "ecur = %"RSYM", ehist[0] = %"RSYM", ehist[0] = %"RSYM,
+                     ecur, hadapt_mem->ehist[0], hadapt_mem->ehist[1]);
+
+  SUNLogger_QueueMsg(ARK_LOGGER, SUN_LOGLEVEL_DEBUG,
+                     "ARKODE::arkAdapt", "step-history",
+                     "hcur = %"RSYM", hhist[0] = %"RSYM", hhist[0] = %"RSYM,
+                     hcur, hadapt_mem->hhist[0], hadapt_mem->hhist[1]);
+
+  SUNLogger_QueueMsg(ARK_LOGGER, SUN_LOGLEVEL_DEBUG,
```

Comment:
It would be good to add a CI test that loops over the different logging levels and just builds the code to catch logging level specific errors like this one
```suggestion
```

Reply:
I agree, but I think this needs to be done later in a new PR.

Reply:
The CI test can be added later but this line needs to be removed to compile with debug logging.

Reply:
Ah, I see now. I quickly tweaked one of the GitHub action workflows to cycle through the logging options.